src/Results/Results.js

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from 'react';
+import Deputado from '../components/Deputado/Deputado';
+import ProgressBar from '../components/ProgressBar/ProgressBar';
+import './Results.scss';
+
+class Results extends Component {
+  onClickShare(deputado) {
+    console.log(deputado);
+  }
+
+  onClickPlus(deputado) {
+    console.log(deputado);
+  }
+
+  renderBotaoVerMais() {
+    const { temMaisRegistros = false } = this.props;
+
+    if(temMaisRegistros) {
+      return (
+        <div className='barra-ver-mais'>
+          <button className='ver-mais'>Ver mais</button>
+        </div>
+      );
+    }
+  }
+
+  renderCandidatos() {
+    const { candidatos = [] } = this.props;
```

Comment:
Acho que aqui ficaria melhor se for feito com `defaultProps` do component. Dessa forma o valor default se aplica para todos os casos.

Reply:
Boa! eu vou botar ele no `defaultProps` sim, não conhecia essa não, valeu :smiley_cat: